src/integration-test/java/io/aiven/kafka/connect/transforms/TopicFromValueSchemaIntegrationTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2019 Aiven Oy
```

Comment:
nit: 
```suggestion
 * Copyright 2023 Aiven Oy
```

Applies to other files too.